xml/System.IO.Compression/BrotliEncoder.xml

```diff
@@ -142,10 +149,10 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, a span of bytes where the compressed data will be stored.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes that were written to <paramref name="destination" />.</param>
+        <summary>Compresses an empty read-only span of bytes into its destination, which ensures that output is produced for all the processed input. An actual flush is performed when the source is depleted and there is enough space in the destination for the remaining data.</summary>
```

Comment:
`Compresses an empty read-only span of bytes into its destination`

This is an implementation detail that is not relevant to the caller (and hence shouldn't be documented).